spark/src/main/scala/geotrellis/spark/io/cog/package.scala

```diff
@@ -26,4 +28,12 @@ package object cog extends Implicits {
         extent.ymax - layout.cellheight / 2
       )
   }
+
+  implicit class withMappedKeyIndexesMethods[K](mappedKeyIndexes: Map[ZoomRange, KeyIndex[K]]) {
```

Comment:
Ah, that was for the `readKeyIndex` method https://github.com/jbouffard/geotrellis/blob/44197c71deeab7a5f052fc381500b0b1f9c03d7a/spark/src/main/scala/geotrellis/spark/io/AttributeStore.scala#L122 I don't remember why, but I wasn't able to get it to work.

I didn't see `zoomRangeInfoFor`, so it might be worth just removing this.